sqlQueries/nlstCohort.sql

```diff
@@ -72,7 +76,10 @@
     WHERE
       -- Filter for CT images in the NLST collection that are not localizers 
       --and removing the transfer syntax ids that require additional processing (decompression before passing to dcm2niix)
-      collection_id = 'nlst' AND Modality = 'CT' AND axes = 2 AND axis1 = 0 AND axis2 = 1 AND 'LOCALIZER' NOT IN UNNEST(ImageType) AND TransferSyntaxUID NOT IN ( '1.2.840.10008.1.2.4.70','1.2.840.10008.1.2.4.51')
+      collection_id = 'nlst' AND Modality = 'CT' AND axes = 2 AND axis1 = 0 AND axis2 = 1 
+      AND SeriesInstanceUID not in (Select SeriesInstanceuID from  `bigquery-public-data.idc_v16.dicom_all` bid, bid.ImageType image_type where image_type='LOCALIZER' OR  TransferSyntaxUID  IN ( '1.2.840.10008.1.2.4.70','1.2.840.10008.1.2.4.51'))
```

Comment:
Let's switch to using `v17` (alternatively - and maybe it's better - update to v17 can be done as a separate PR to fix it in all of the places).

Reply:
there is one query that queries idc, so I fixed the idc version to v17